infra/build/src/bin/publish.ts

```diff
@@ -77,9 +77,14 @@ const PRERELEASE_ID = `.${Date.now()}`
 const FILES = {
   README: readFile(join(Paths.CLI, 'README.md')),
   LICENSE: readFile(join(Paths.CLI, 'LICENSE')),
-  PACKAGE_JSON: readPackageJson(
-    join(Paths.CLI, 'package.json'),
-  ) as Bundle,
+  PACKAGE_JSON: (() => {
+    const pkg = readPackageJson(
+      join(Paths.CLI, 'package.json'),
+    ) as Bundle
+    // Only keep the node engines for publishing
+    pkg.engines = { node: pkg.engines.node }
```

Comment:
this doesn't seem to be working... I'm still seeing `engines.pnpm` references in the built package when doing a dry run (ie. `pnpm -F infra-build exec node dist/esm/bin/publish.js`)

Reply:
*had to rebuild... looks like it's working now